dubbo-common/src/main/java/org/apache/dubbo/rpc/model/ModuleModel.java

```diff
@@ -61,9 +61,6 @@ public ModuleModel(ApplicationModel applicationModel, boolean isInternal) {
         }
 
         initialize();
-        Assert.notNull(serviceRepository, "ModuleServiceRepository can not be null");
-        Assert.notNull(moduleConfigManager, "ModuleConfigManager can not be null");
-        Assert.assertTrue(moduleConfigManager.isInitialized(), "ModuleConfigManager can not be initialized");
```

Comment:
why remove this line?

Reply:
Of course, it can be reserved, but I just want to unify it with `ApplicationModule`.(`ApplicationModule` does not check its `configManager` and  `serviceRepository`)

Reply:
It would be better that add this check to `ApplicationModel`

Reply:
> It would be better that add this check to `ApplicationModel`

done